tests/TestEntry.cpp

```diff
@@ -262,3 +262,167 @@ void TestEntry::testResolveRecursivePlaceholders()
     QCOMPARE(entry5->resolveMultiplePlaceholders(entry5->url()), url);
     QCOMPARE(entry5->resolveMultiplePlaceholders(entry5->title()), QString("title+/some/path+fragment+title"));
 }
+
+void TestEntry::testResolveReferencePlaceholders()
+{
+    Database db;
+    Group* root = db.rootGroup();
+
+    Entry* entry1 = new Entry();
+    entry1->setGroup(root);
+    entry1->setUuid(Uuid::random());
+    entry1->setTitle("Title1");
+    entry1->setUsername("Username1");
+    entry1->setPassword("Password1");
+    entry1->setUrl("Url1");
+    entry1->setNotes("Notes1");
+    entry1->attributes()->set("CustomAttribute1", "CustomAttributeValue1");
+
+    Group* group = new Group();
```

Comment:
Group declared (added to the root group) but never used

Reply:
typo, fixed ;)